lightning/src/util/events.rs

```diff
@@ -617,7 +617,7 @@ pub enum Event {
 		/// [`ChannelManager::create_channel`]: crate::ln::channelmanager::ChannelManager::create_channel
 		/// [`ChannelManager::accept_inbound_channel`]: crate::ln::channelmanager::ChannelManager::accept_inbound_channel
 		/// [`UserConfig::manually_accept_inbound_channels`]: crate::util::config::UserConfig::manually_accept_inbound_channels
-		user_channel_id: u64,
+		user_channel_id: u128,
```

Comment:
I assume we can avoid the serialization in two `u64` in the case of `ChannelReady`, as this PR hopefully also lands in before the release of 0.0.113, i.e., it will be `u128` from 'the beginning'?